### DISCUSSION TITLE
Parallel 1P Common Low

### DISCUSSION THREAD

**Discussion by ThatOldNerd** [darthcloud/BlueRetro#606] [General]
I am working on an adapter for the Colecovision/Adam and building it around the Parallel 1P (NeoGeo) adapter.  My issue is the NeoGeo's outputs are held high, driven Low, my design is to drive CD4066 chips, and I need to hold low and drive high.  Is there an alternate configuration, or a way to adapt it.  Thank you in advance.

**Comment by darthcloud** [comment]
Follow the push-pull version https://github.com/darthcloud/BlueRetroHW/blob/master/DIY/Para1P_PP.pdf

Replace the 74AHCT125N with a Hex inverter like 74AHCT04N or anything equivalent.

https://www.digikey.com/en/products/detail/texas-instruments/SN74AHCT04N/375913?s=N4IgjCBcoGwJxVAYygMwIYBsDOBTANCAPZQDaIALGGABxwDsIAuoQA4AuUIAyuwE4BLAHYBzEAF9xhAExkQ9CgEEAEgGEAKgAYKzcUA

The pinout is different and you got 6 gate rather than four, but you should be able to figure out how to wire it.

**Reply by ThatOldNerd** [reply]
Thank you I will update my circuit.  I have a working adapter from the NeoGeo and use the 74hc04s to modify the signals.  I was just trying to reduce chip count by direct driving the 4066 chips.
